xml/System.Diagnostics/DiagnosticSource.xml

```diff
@@ -139,8 +139,8 @@ If there is expensive setup for the notification, you can call this method and p
         <Parameter Name="payload" Type="System.Object" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="activity">To be added.</param>
-        <param name="payload">To be added.</param>
+        <param name="activity">The activity affected by an external event.</param>
+        <param name="payload">An object that represents the outgoing request.</param>
         <summary>Transfers state from an activity to some event or operation, such as an outgoing HTTP request, that will occur outside the process.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
```

Comment:
There is a typo in line 148, unrelated to this PR but lets get it fixed now if we can:

> If an *instumentation*

Should be:

> If an **instrumentation**

Reply:
Line 150 similar case:

> To the extent possible, this should be done by the *instrumenation*

Should be:

> To the extent possible, this should be done by the **instrumentation**